.eslintrc.js

```diff
@@ -1,10 +1,10 @@
-/**
- * These rules enforce the Hack Reactor Style Guide
- *
- * Visit this repo for more information:
- *   https://github.com/reactorcore/eslint-config-hackreactor
- */
-
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
-};
\ No newline at end of file
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: 'airbnb-base',
```

Comment:
Much nicer! I'll be sure to clear my .eslintrc.js file of any superfluous references.